.github/workflows/github-users.json

```diff
@@ -0,0 +1,23 @@
+{
+  "filecoin-project": {
+    "user": "web3-bot-filecoin"
+  },
+  "filecoin-shipyard": {
+    "user": "web3-bot"
+  },
+  "ipfs": {
+    "user": "web3-bot-ipfs"
+  },
+  "ipfs-cluster": {
+    "user": "web3-bot"
```

Comment:
naively, wouldn't this be web3-bot-ipfs?

Reply:
That depends on how Andy configures it. At the moment there aren't any changes. Later on, it's debatable if this should be web3-bot-ipfs or web3-bot-ipfs-cluster. For consistency, I assume we'll probably just create a new user.